pkgs/by-name/gi/gitfetch/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  gh,
+  nix-update-script,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "gitfetch";
+  version = "1.2.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Matars";
+    repo = "gitfetch";
+    tag = "v${version}";
+    hash = "sha256-2cOfVv/snhluazyjwDuHEbbMq3cK+bsKYnnRmby0JDo=";
+  };
+
+  dependencies = with python3Packages; [
+    requests
+    readchar
+  ];
+
+  nativeBuildInputs =
+    with python3Packages;
+    [
+      setuptoolsBuildHook
+      wheel
+    ]
+    ++ [
+      gh
+    ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "A neofetch-style CLI tool for git provider statistics";
+    homepage = "https://github.com/Matars/gitfetch";
+    mainProgram = "gitfetch";
+    license = lib.licenses.gpl2;
```

Comment:
Forgot we are not supposed to use the base license definition anymore, should be `lib.licenses.gpl2Only` I believe ([LICENSE](https://github.com/Matars/gitfetch/blob/main/LICENSE) and [README](https://github.com/Matars/gitfetch/tree/main?tab=readme-ov-file#license)).